build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/CreateAndroidEmulator.cs

```diff
@@ -108,7 +109,7 @@ void Exec (string android, string arguments, DataReceivedEventHandler stderr = n
 				UseShellExecute         = false,
 				RedirectStandardInput   = true,
 				RedirectStandardOutput  = false,
-				RedirectStandardError   = false,
+				RedirectStandardError   = true,
```

Comment:
...how did this ever work before?  Below (old line 128) we subscribe to `p.ErrorDataReceived += stderr`, but without `RedirectStandardError=true`, `p.ErrorDataReceived` should never be raised...right?

Reply:
yep, that's correct, it would never be called. Unfortunately, the loop checking for `HasExited` isn't ideal either, as described, but we can't do much about it. `avdmanager` output is.... messy and parsing it makes no real sense. I wonder if we want to log stdout here too, btw?